### DISCUSSION TITLE
It is possible to color the selection box of a highlighted file in Windows 11 dark theme?

### DISCUSSION THREAD

**Discussion by Dhyfer1** [Abdelrhman-AK/WinPaletter#152] [General]
Hi @Abdelrhman-AK 

I use the dark theme in Windows 11 and I would like to know if it is possible for Winpaletter to colorize the selection box of a highlighted file in file explorer.

[AccentColorizer ](https://github.com/krlvm/AccentColorizer) as you can see in the screenshots it can do it when Windows is in light theme, but it doesn't work with dark theme. So Winpaletter can achieve this?

**Comment by Abdelrhman-AK** [comment]
I'm sorry, I can't implement this as I don't have knowledge about visual styles patching.

You can request this feature in AccentColorizer repository.

**Reply by Dhyfer1** [reply]
> I'm sorry, I can't implement this as I don't have knowledge about visual styles patching.

Oh, what a pity. :( I hope one day Winpaletter can do it if you are interested in patching visual styles

> You can request this feature in AccentColorizer repository.

I had already [informed](https://github.com/krlvm/AccentColorizer/issues/10) the author of AccentColorizer about this, but he says that he can't do anything about it. :(

Anyway, thanks for answering.

**Reply by Abdelrhman-AK** [reply]
You're welcome!
